docs/TROUBLESHOOTING.md

```diff
@@ -102,6 +102,16 @@ ControllerMode = dual
 JustWorksRepairing = confirm
 ```
 
+#### Xbox Wireless controller
+
+The newest wireless controllers from Microsoft (Xbox One and Xbox Series X/S) are known to cause a reconnect loop and not pairing with Bluez. There are some specific workarounds :
+
+- Plug your controller to a Windows 10 computer via a USB cord. Download the [xbox accessories application](https://xbox.com/accessories-app). When launching the app, it should ask you to update the firmware of the controller. When it's done, your controller should work just fine with any Linux system.
+- If it didn't work, you can try these two workarounds :
+  - Use a Windows 10 computer *on the same Bluetooth adapter* to pair with the controller. It must absolutly be on the same Bluetooth adapter, ie the same computer (can be inside a virtual machine with Bluetooth passthrough) if it's an internal Bluetooth adapter, or the same Bluetooth dongle. Then, you can get the pairing keys and install them within your Linux Bluetooth system.
+  - Update to a newer kernel. Kernel 5.13 and higher might have patched a fix.
+  - Use a different Bluetooth stack. Xbox controllers work fine with Fluoriden the bluetooth stack from Android. Sadly, it's hard to install on another Linux, and Bluez is the only stack easily provided on most Linux distributions.
+- If none of these options worked, or you can't try them, then the only solution is to plug the controller using a USB cord.
```

Comment:
Since this causes confusion all the time, please add a note here that using a USB cable will no longer use the xpadneo driver then. I will be adding USB support soon to xpadneo but it's not quite there yet.